content/cli.md

```diff
@@ -508,7 +554,25 @@ applications:
 ```
 {{</tab>}}
 {{<tab tabNum="3">}}
-TODO: Updating in Java
+
+Create `manifest.yaml` in your project's root folder with the following content:
+```yaml
+---
+applications:
+- name: mysample
+  memory: 1G
```

Comment:
Does it really need 1G? I've reduced the Python examples to 128M and they run fine - allows more apps deployed in parallel with our 2G quota.